core/templates/dev/head/domain/feedback_thread/FeedbackThreadObjectFactory.js

```diff
@@ -29,6 +29,11 @@ oppia.factory('FeedbackThreadObjectFactory', [function() {
     this.messageCount = messageCount;
     this.stateName = stateName;
     this.threadId = threadId;
+    this.messages = null;
```

Comment:
When using the FeedbackThread object, do you ever include a case for "if messages === null" anywhere?

If not, then safer to set it to [], I think.

Reply:
That would be better! Changed it, thanks!